src/styles/blocks/card.scss

```
@@ -0,0 +1,81 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  color: $main-text-color;
+
+  &__product-image {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__product-name {
+    margin-bottom: 4px;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    font-size: $service-info-text-size;
+    font-weight: $service-info-text-weight;
+    color: $product-code-text-color;
+  }
+
+  &__review {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-size: $service-info-text-size;
+    font-weight: $service-info-text-weight;
+    color: $main-text-color;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    font-weight: 400;
+    color: $price-text-color;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    text-align: right;
+    color: #000;
+  }
```

Use scss nesting, fix it everywhere
```suggestion
  &__price {
    margin-bottom: 16px;
    display: flex;
    justify-content: space-between;
    align-items: center;
    
    &-text {
      // write styles here
    }
    
    &-value {
      // write styles here
    }
  }
```